### ISSUE TITLE
shown.bs.dropdown & hidden.bs.dropdown are not triggered

### ISSUE THREAD

**Issue by golear91** [CWSpear/bootstrap-hover-dropdown#152]
http://getbootstrap.com/javascript/#dropdowns-events

It seems as though these events are not being triggered by the dropdown that is a result of hovering over using this library. (the other two, show and hide are triggered)

**Comment by holtkamp** [comment]
So to be clear:
- triggered
  - show.bs.dropdown	This event fires immediately when the show instance method is called.
  - hide.bs.dropdown	This event is fired immediately when the hide instance method has been called.
- **not** triggered
  - shown.bs.dropdown	This event is fired when the dropdown has been made visible to the user (will wait for CSS transitions, to complete).
  - hidden.bs.dropdown This event is fired when the dropdown has finished being hidden from the user (will wait for CSS transitions, to complete).

It seems [anticipated for](https://github.com/CWSpear/bootstrap-hover-dropdown/blob/1b04355723796bc58bdb202ad2a494233264b5d9/bootstrap-hover-dropdown.js#L43-L44) but not used / out of scope for this component? @goleary can you provide a https://jsfiddle.net demonstrating the behaviour?

**Comment by goleary** [comment]
you are correct. seeing as it shows in the code that this is by design (?)I'd rather not spend my time providing the fiddle (I added this a year ago and have since not been working with the library so it's more trouble than it is worth to me at this point).

I don't think it makes sense for the events to not work with this library but again i'm not a user so not worth investing the time for me.
